lib/rihanna_ui_web/controllers/jobs_controller.ex

```diff
@@ -26,4 +26,11 @@ defmodule RihannaUIWeb.JobsController do
     end
     |> redirect(to: "/failed")
   end
+
+  def mutate(conn, %{"action" => "retry_all"}) do
```

Comment:
This is a bit scizophrenic. I suppose this either belongs in the `case` statement above, or all the `case` actions should be pulled out into function heads.